pkg/tnf/handlers/handler_template/handler.tmpl

```diff
@@ -18,15 +18,10 @@ package {{ .Handlername }}
 
 import (
 )
-//TODO
-// Hostname provides a hostname test implemented using command line tool "hostname".
-/*
+
+//TODO:
 type {{ .Handlername }} struct {
```

Comment:
in here
``
timeout time.Duration
	args    []string
``
Those parameters are always present. You can keep them in the template

Reply:
this kept in template.